heliopy/data/dscovr.py

```diff
@@ -0,0 +1,67 @@
+"""
+Methods for importing data from the DSCOVR.
+
+All data is publically available at
+ftp://spdf.gsfc.nasa.gov/pub/data/dscovr/.
+"""
+import pathlib as path
+import os
+import pandas as pd
+import astropy.units as u
+import datetime as dt
+from collections import OrderedDict
+
+from heliopy.data import util
+from heliopy import config
+
+data_dir = path.Path(config['download_dir'])
+dscovr_dir = data_dir / 'dscovr'
+dscovr_url = 'https://spdf.gsfc.nasa.gov/pub/data/dscovr/'
```

Comment:
👍 👍 👍  I didn't know that DSCOVR data was on SPDF!

Reply:
Here's a really cool list we can use for adding more data sources! https://omniweb.sci.gsfc.nasa.gov/ftpbrowser/